Lecture_18/Homework_18.py

```diff
@@ -0,0 +1,33 @@
+class Bot:
+    def __init__(self, name,):
+        self.name = name
+    def say_name(self, name):
+        print(self.name)
+    def send_message(self, message):
+        print(message)
+
+
+class TelegramBot(Bot):
+    def __init__(self, name, url=None, chat_id=None):
+        super().__init__(name)
+        self.url = url
+        self.chat_id = chat_id
+    def send_massage(self, message):
+        super().send_message()
+        return TelegramBot(self.send_message())
```

Comment:
here method  `send_message` from `Bot` should be rewritten
```suggestion
        print(f'{self.name} bot says {message} to chat {self._chat_id} using {self._url}')
```